v3/control.go

```diff
@@ -911,6 +929,9 @@ func (c *ControlServerSideSorting) GetControlType() string {
 }
 
 func NewControlServerSideSorting(value *ber.Packet) (*ControlServerSideSorting, error) {
+	if value == nil || len(value.Children) < 2 {
+		return new(ControlServerSideSorting), nil
```

Comment:
This would  satisfy your scanner, but would result in a faulty message due to the controlValue not having the necesarry fields for encoding the message. 

https://github.com/go-ldap/ldap/blob/82fef14a4694eeccea5f6dd33b0c0ab2f17f29ce/v3/control.go#L948-L979

I suggest to disregard this particular change. The function is never called from a message processor but only by a developer.